tests/kuttl/tests/change_keystone_config/03-change_keystone_config.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc patch keystoneapi -n openstack keystone --type='json' -p='[{"op": "replace", "path": "/spec/defaultConfigOverwrite", "value":{"keystone.conf": "debug = true"}}]'
```

Comment:
I'm wondering if `spec.customServiceConfig` would be a better candidate for this test as the above command actually removes all the default configuration of the keystone service the operator creates and only leaves a keystone.conf with debug = True in it and that is not how we would like to enable debug logging in a real deployment.

Reply:
I tried to use `customServiceConfig` but I did not manage to get the configuration applied. Could you give me some indication on how to use it? Also, as @abays pointed out in another comment, the default value of  `customServiceConfig` already includes by default `debug = True`, but the logs do not show debug messages by default.

Reply:
If the default is debug = True then either 
* a) you can deploy with a different yaml to have it not set by default
OR
* b) the test can be reversed. Check that there are DEBUG logs after deploy, set debug = False in customServiceConfig and check if the DEBUG logs does not appear any more.

Reply:
I think the easiest thing is to do `b`.  

Currently in our testing, it does not look like the `spec.customServiceConfig` is working...but I think the problem might be on the OpenStack side as opposed to something we're doing wrong in the OCP operator.  I do see `/etc/keystone/keystone.conf.d/custom.conf` appearing in the Keystone pod with the expected `debug = true` setting under `[DEFAULT]`.  The Keystone service does not seem to be honoring it, however.